### ISSUE TITLE
Unclosed connector

### ISSUE THREAD

**Issue by joejuzl** [elastic/elasticsearch-py-async#43]
Hi,

I am getting an error message when using the client.

**Error message**
```
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7fb608150c88>, 170770.66)]']
connector: <aiohttp.connector.TCPConnector object at 0x7fb608128588>
```

**My code:**
```
elastic_client = AsyncElasticsearch([f'{ES_USER}:{ES_PASSWORD}@{ES_HOST}'], loop=eloop)
await elastic_client.index(index=es_index(doc_type), doc_type=doc_type, body=activity)
elastic_client.transport.close()
```

This is called in an `aiohttp.web.Application` middleware. The loop is created when the app is created using `asyncio.get_event_loop()`.

The request still succeeds, but the message is noisy.

I have tried with and without the line `elastic_client.transport.close()` to no avail. And sometimes (sporadically) with it there I get the error message: 
```
/usr/local/lib/python3.6/site-packages/elasticsearch/connection_pool.py:255: RuntimeWarning: coroutine 'ClientSession.close' was never awaited
```
However `await` the same line throws a different error.

**Versions:**

python ==3.6.4
elasticsearch-async ==6.0.0
aiohttp ==3.1.3

Am I doing something wrong, or is this a bug?

Thanks!

**Comment by levchik** [comment]
I'm experiencing the same issues. After server (we use sanic) stops, massive traceback occurs about error in closing session. I saw #39 PR and I think this is great and should be merged as soon as possible.

**Comment by yarara** [comment]
Hello. Who can publish this update to pypi?

**Comment by gothraven** [comment]
I'm still having the same issue in 2019, the pypi update is not done yet ?

**Comment by udfalkso** [comment]
Just adding my +1 here.  Any news on this?  Thanks!

**Comment by konpap94** [comment]
Same problem here!!!

**Comment by deansg** [comment]
It seems to me that there is already code fixing this issue in the AsyncElasticsearch class (the await self.transport.close() line in the aexit method). However, this code is from April 2019, while the latest release of the package is from August 2018. If a new release can be made that includes the bug fix, it'll be great. For now, I just use a try-finally with the AsyncElasticsearch object on my own

**Comment by gothraven** [comment]
@deansg its what we've been asking for, for the last two years !

**Comment by rappongy** [comment]
Still facing it.

```
python 3.7
aiohttp 3.6.2
elasticsearch 7.6.0
elasticsearch_async 6.2.0
```

**Comment by sethmlarson** [comment]
This issue is fixed in `elasticsearch[async]` 7.8.0, see https://github.com/elastic/elasticsearch-py-async/issues/81 for more information.
